disco_aws_automation/disco_rds.py

```diff
@@ -222,7 +222,7 @@ def modify_db_instance(self, instance_params, apply_immediately=True):
         logger.info("Updating RDS cluster %s", instance_params["DBInstanceIdentifier"])
         params = RDS.delete_keys(instance_params, [
             "Engine", "LicenseModel", "DBSubnetGroupName", "PubliclyAccessible",
-            "MasterUsername", "Port", "CharacterSetName", "StorageEncrypted"])
+            "MasterUsername", "Port", "CharacterSetName", "StorageEncrypted", "Tags"])
```

Comment:
Have to delete the `Tags` key from `instance_params` because `self.client.modify_db_instance(..)` doesn't take it.